### PR TITLE
Feature/add endpoint personal location

### PR DESCRIPTION
**Pull request by Yojuks**

### CODE REVIEW

**Review by steadyGuy — APPROVED** [review]
Swagger docs for the endpoint needed

**Review by romawht22 — APPROVED** [review]

File: backend/models/Locations.ts [hunk 1]
```diff
@@ -31,19 +31,19 @@ const schema = new mongoose.Schema({
...
         type: Array,
-        default: []
+        default: [],
       },
       author: String,
       cost: String,
-    }
-  }
+    },
+  },
```
Comment (6mahdapihka9) [inline backend/models/Locations.ts:46]: are these commas necessary?

File: backend/routes/index.ts [hunk 2]
```diff
@@ -7,17 +7,24 @@ import passport from "../libs/passport";
...
+  "/get_profile_location",
+  passport.authenticate("jwt", { session: false }),
+  UserController.postUserLocation
+);
 router.get(
-  "/protected-route",
-  passport.authenticate("jwt", { session: false })
+  "/userData/:_id",
+  passport.authenticate("jwt", { session: false }),
+  UserController.getUserData
```
Comment (6mahdapihka9) [inline backend/routes/index.ts]: this endpoint changed in last develop update

**Review by 6mahdapihka9 — APPROVED** [review]
pull the latest develop update, because some conflicts may occur

**Review by 6mahdapihka9 — CHANGES_REQUESTED** [review]
I'm pretty sure that you should checkout develop

File: backend/models/Locations.ts (hunk 1 above)
Comment (Yojuks) [reply backend/models/Locations.ts:46]: there were after auto save in vscode

File: backend/routes/index.ts (hunk 2 above)
Comment (Yojuks) [reply backend/routes/index.ts]: ok - thanks

**Review by 6mahdapihka9 — APPROVED** [review]

**Pull request merged**